scripts/publish-packages.sh

```diff
@@ -0,0 +1,19 @@
+
+#!/bin/bash
+
+set -e
+
+npm i
+
+npx playwright install
+
+# Ensure that all public packages are in this list,
+# otherwise they won't get built/linted/tested before being published
+npx turbo run build lint typecheck test \
```

Comment:
nice, is there a reason we dont do `verify + build`?

Reply:
Not all of our packages have a `verify` script, but maybe I should change that